eng/scripts/Show-FailureLogs.ps1

```diff
@@ -6,7 +6,8 @@
 # sensitive information.
 
 $logFiles = Get-ChildItem -Recurse -Filter *.log
-$filteredLogs = $logFiles.Where({ $_.Name -in ('vcpkg-bootstrap.log', 'vcpkg-manifest-install.log') })
+$vcpkgLogFileNames = ('vcpkg-bootstrap.log', 'vcpkg-manifest-install.log')
```

Comment:
Does this list cover all the interesting logs from vcpkg/CMake generate that we care about?

Reply:
As far as I can quickly tell from my memory, yes.

Reply:
As I am investigating an issue with vcpkg, I actually may end up with finding more good files to include here, for diagnosability. I.e. build command lines, and so on., to gather all the necessary info that will end up being useful to solving the CI bug that we are seeing, because it only repos in our CI.